cmd/ethrex_replay/src/run.rs

```diff
@@ -1,45 +1,44 @@
 use crate::cache::Cache;
-use ethrex_common::types::ELASTICITY_MULTIPLIER;
+use ethrex_common::types::{AccountUpdate, Receipt, ELASTICITY_MULTIPLIER};
+use ethrex_levm::db::{gen_db::GeneralizedDatabase, CacheDB};
+use ethrex_vm::{backends::levm::LEVM, Evm};
+use eyre::Ok;
+use std::sync::Arc;
 use zkvm_interface::io::ProgramInput;
 
 pub async fn exec(cache: Cache) -> eyre::Result<String> {
```

Comment:
Can we refactor this function's return to `eyre::Result<()>`? This way we'll get rid of the `Ok("".to_string())`.